backend/tests/integration/proposed_change/test_proposed_change_conflict.py

```diff
@@ -199,7 +199,7 @@ async def test_conflict_pipeline(
         john = await NodeManager.get_one_by_default_filter(db=db, id="John", kind=TestKind.PERSON)
         # The value of the description should match that of the source branch that was selected
         # as the branch to keep in the data conflict
-        assert john.description.value == "Oh boy"  # type: ignore[attr-defined]
+        assert john.description.value == "Oh boy"  # type: ignore[attr-defined, union-attr]
```

Comment:
The idea is that we'll generate additional protocols for the test classes (we have a small number now) then when we start to use those within the tests we can phase out these type ignore statements.